docs/validators/changeset.rst

```diff
@@ -25,6 +25,10 @@ Change set
       max:
          count: 2 # max number of files in a PR
          message: 'Custom message...'
+      files:
+         added: true
```

Comment:
for consistency sake, would you please add the `default` value of each field? i.e
`added: true #default : true, if false, added files will not counted toward changeset`